src/MasterSite.php

```diff
@@ -195,6 +195,16 @@ function ( $block_content, $block ) {
 		add_action( 'admin_notices', [ $this, 'show_dashboard_notice' ] );
 		add_action( 'wp_ajax_dismiss_dashboard_notice', [ $this, 'dismiss_dashboard_notice' ] );
 		add_filter( 'timber/twig', [ $this, 'p4_optimize_img_url' ] );
+
+		// Pin the ElasticPress to v3.4 search algorithm.
+		if ( \is_plugin_active( 'elasticpress/elasticpress.php' ) ) {
```

Comment:
I think we can get around the issue by simply not doing the `is_plugin_active` check. The `ep_search_algorithm_version` filter is exclusive to the plugin, adding it without the plugin loaded shouldn't cause any issues.